tensorflow-sys/build.rs

```diff
@@ -0,0 +1,49 @@
+extern crate pkg_config;
+
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::{env, fs};
+
+const LIBRARY: &'static str = "tensorflow";
+const REPOSITORY: &'static str = "https://github.com/tensorflow/tensorflow.git";
+const TARGET: &'static str = "libtensorflow.so";
+const VERSION: &'static str = "0.9.0";
+
+macro_rules! get(($name:expr) => (ok!(env::var($name))));
+macro_rules! ok(($expression:expr) => ($expression.unwrap()));
+
+fn main() {
+    if pkg_config::find_library(LIBRARY).is_ok() {
+        return;
+    }
+
+    let output = PathBuf::from(&get!("OUT_DIR"));
+    if !output.join(TARGET).exists() {
+        let source = PathBuf::from(&get!("CARGO_MANIFEST_DIR")).join("target/source");
+        if !Path::new(&source.join(".git")).exists() {
+            run("git", |command| command.arg("clone")
+                                        .arg(format!("--branch=v{}", VERSION))
+                                        .arg("--recursive")
+                                        .arg(REPOSITORY)
+                                        .arg(&source));
+        }
+        run("./configure", |command| command.current_dir(&source));
+        run("bazel", |command| command.current_dir(&source)
+                                      .arg("build")
+                                      .arg(format!("--jobs={}", get!("NUM_JOBS")))
+                                      .arg("--compilation_mode=opt")
+                                      .arg(format!("{}:{}", LIBRARY, TARGET)));
+        let source = source.join("bazel-out/local-opt/bin/tensorflow");
```

Comment:
If you use "bazel-bin/tensorflow", then it doesn't include the build configuration ("local-opt").